test.py

```diff
@@ -0,0 +1,157 @@
+import json
+import logging
+import types
+import webapp2
+from datetime import datetime, timedelta
```

Comment:
fwiw, I've gotten away from importing inner libraries because they mess up the code too much vs the value of explicitly seeing which lib a method comes from... when I need to import inners to reduce verbosity, I pick new names.

e.g. in BBF, we "import persistentcache as pc" consistently, so code readers can always know that pc.something is from persistentcache.